pyvista/plotting/tools.py

```diff
@@ -180,8 +180,57 @@ def normalize(x, minimum=None, maximum=None):
     return (x - minimum) / (maximum - minimum)
 
 
-def opacity_transfer_function(mapping, n_colors, interpolate=True):
-    """Get the opacity transfer function results: range from 0 to 255."""
+def opacity_transfer_function(mapping, n_colors, interpolate=True,
+                              kind='quadratic'):
+    """Get the opacity transfer function for a mapping.
+
+    These values will map on to a scalar bar range and thus the number of
+    colors (``n_colors``) must correspond to the number of colors in the color
+    mapping that these opacities are associated to.
+
+    If interpolatig, ``scipy.interpolate.interp1d`` is used if available,
```

Comment:
"interpolatig" seems like a spelling error.

Reply:
Hah thanks, ran codespell and fixed a lot of other typos as well